tools/update-tester/src/testrunner.rs

```diff
@@ -360,7 +363,10 @@ impl TestClient {
                 test_client.validate_stable_objects_exist();
 
                 test_client
-                    .simple_query("SET TIME ZONE 'UTC';")
+                    .simple_query(&format!(
+                        r#"ALTER DATABASE "{}" SET timezone TO 'UTC';"#,
+                        &test_db_name
+                    ))
```

Comment:
This shouldn't be necessary here.  This persistently sets the default for the database, so just doing it when the database is created should be sufficient.